plugins/repository-hdfs/build.gradle

```diff
@@ -68,6 +71,61 @@ task hdfsFixture(type: org.elasticsearch.gradle.test.AntFixture) {
        baseDir
 }
 
+// MIT Kerberos Vagrant Testing Fixture
+def box = "krb5kdc"
+def vagrantEnvVars = [
+        'VAGRANT_CWD'           : "${project(':test:fixtures:krb5kdc-fixture').projectDir}",
+        'VAGRANT_VAGRANTFILE'   : 'Vagrantfile',
+        'VAGRANT_PROJECT_DIR'   : "${project(':test:fixtures:krb5kdc-fixture').projectDir}"
+]
+
+task krb5kdcUpdate(type: org.elasticsearch.gradle.vagrant.VagrantCommandTask) {
+  command 'box'
+  subcommand 'update'
+  boxName box
+  environmentVars vagrantEnvVars
+}
+
+Fixture krb5kdcFixture = project.tasks.create("krb5kdcFixture", org.elasticsearch.gradle.test.VagrantFixture) {
```

Comment:
I don't think there is a need for this local? Creating the task will add this task to the project, which will resolve correctly when doing dependsOn later.